src/common/options/osd.yaml.in

```diff
@@ -287,7 +287,7 @@ options:
   type: float
```

Comment:
This commit removes ~2.3kloc and adds ~4.4kloc.  Is there really no way to split this up?  The commit message needs a very short (<= 4 sentence) that explains the *goal* of the "redesign".  I'd also encourage using a word other than "redesigned" as it gives no useful clue to a reviewer or future reader as to what you are trying to achieve.